packages/backend/server/src/plugins/indexer/config.ts

```diff
@@ -58,4 +62,14 @@ defineModuleConfig('indexer', {
     default: '',
     env: ['AFFINE_INDEXER_SEARCH_PASSWORD', 'string'],
   },
+  'autoIndexState.lastIndexedWorkspaceSid': {
```

Comment:
this is not suitable to be put in config

Reply:
A new `indexed` field is added to the workspaces table to indicate whether the index has been created. And we can also reset it to rebuild the index